frontend/app/services/personal-information-service.server.ts

```diff
@@ -151,41 +246,44 @@ function createPersonalInformationService() {
+      clientNumber: personalInformationApiResponse.BenefitApplication.Applicant?.ClientIdentification?.filter((clientInfoDto) => clientInfoDto.IdentificationCategoryText == 'Client Number').at(0)?.IdentificationID,
+      birthDate: personalInformationApiResponse.BenefitApplication.Applicant?.PersonBirthDate.dateTime,
+      firstName: personalInformationApiResponse.BenefitApplication.Applicant?.PersonName?.at(0)?.PersonGivenName?.at(0),
+      lastName: personalInformationApiResponse.BenefitApplication.Applicant?.PersonName?.at(0)?.PersonSurName,
+      emailAddress: personalInformationApiResponse.BenefitApplication.Applicant?.PersonContactInformation?.EmailAddress.EmailAddressID,
+      maritalStatus: personalInformationApiResponse.BenefitApplication.Applicant?.PersonMaritalStatus.StatusCode.ReferenceDataID,
       homeAddress: homeAddressList
         ?.map((aHomeAddress) => ({
           streetName: aHomeAddress.AddressStreet.StreetName,
           cityName: aHomeAddress.AddressCityName,
-          provinceName: aHomeAddress.AddressProvince.ProvinceName,
-          countryName: aHomeAddress.AddressCountry.CountryName,
-          countryCode: aHomeAddress.AddressCountry.CountryCode,
+          provinceCode: aHomeAddress.AddressProvince.ProvinceCode.ReferenceDataID,
+          countryCode: aHomeAddress.AddressCountry.CountryCode.ReferenceDataID,
           postalCode: aHomeAddress.AddressPostalCode,
         }))
         .at(0),
       mailingAddress: mailingAddressList
         ?.map((aMailingAddress) => ({
           streetName: aMailingAddress.AddressStreet.StreetName,
           cityName: aMailingAddress.AddressCityName,
-          provinceName: aMailingAddress.AddressProvince.ProvinceName,
-          countryName: aMailingAddress.AddressCountry.CountryName,
-          countryCode: aMailingAddress.AddressCountry.CountryCode,
+          provinceCode: aMailingAddress.AddressProvince.ProvinceCode.ReferenceDataID,
+          countryCode: aMailingAddress.AddressCountry.CountryCode.ReferenceDataID,
           postalCode: aMailingAddress.AddressPostalCode,
         }))
         .at(0),
-      language: {
-        languageCode: personalInformationApiResponse.Client.PersonLanguage?.LanguageCode.ReferenceDataName,
-        preferredIndicator: personalInformationApiResponse.Client.PersonLanguage?.PreferredIndicator,
-      },
+
+      primaryTelephoneNumber: personalInformationApiResponse.BenefitApplication.Applicant?.PersonContactInformation?.TelephoneNumber.find((phoneNumber) => phoneNumber.TelephoneNumberCategoryCode.ReferenceDataName == 'Primary')?.FullTelephoneNumber
+        .TelephoneNumberFullID,
+      alternateTelephoneNumber: personalInformationApiResponse.BenefitApplication.Applicant?.PersonContactInformation?.TelephoneNumber.find((phoneNumber) => phoneNumber.TelephoneNumberCategoryCode.ReferenceDataName == 'Alternate')?.FullTelephoneNumber
+        .TelephoneNumberFullID,
```

Comment:
I noticed there are a few occurrences of `==` being used in this file that should all be replaced with `===`.

```suggestion
      primaryTelephoneNumber: personalInformationApiResponse.BenefitApplication.Applicant?.PersonContactInformation?.TelephoneNumber.find((phoneNumber) => phoneNumber.TelephoneNumberCategoryCode.ReferenceDataName === 'Primary')?.FullTelephoneNumber
        .TelephoneNumberFullID,
      alternateTelephoneNumber: personalInformationApiResponse.BenefitApplication.Applicant?.PersonContactInformation?.TelephoneNumber.find((phoneNumber) => phoneNumber.TelephoneNumberCategoryCode.ReferenceDataName === 'Alternate')?.FullTelephoneNumber
        .TelephoneNumberFullID,
```

Reply:
Updated: 3489b449c8a8d57369f2afe860d035d0deaf0e1e